flutter/README.md

```diff
@@ -0,0 +1,79 @@
+# fluentui_system_icons
```

Comment:
This should probably be stylized more readably as FluentUI System Icons

Reply:
or just Fluent System Icons, as that's what the iOS/Android libraries call it

Reply:
This should be "Documentation" to match what Android/iOS have.